placenames.mss

```diff
@@ -4,34 +4,22 @@
 @state-labels: desaturate(darken(@admin-boundaries, 5%), 20%);
 
 .country {
-  [zoom >= 3][way_pixels > 1000][way_pixels < 360000] {
+  [zoom >= 3][way_pixels > 1000][way_pixels < 192000] {
     text-name: "[name]";
-    text-size: 10;
-    text-wrap-width: 35; // 3.5 em
-    text-line-spacing: -1.5; // -0.15 em
+    text-size: @landcover-font-size;
+    text-wrap-width: @landcover-wrap-width-size;
+    text-line-spacing: @landcover-line-spacing-size;
     text-margin: 7.0; // 0.7 em
-    [zoom >= 4] {
-      text-size: 11;
-      text-wrap-width: 40; // 3.6 em
-      text-line-spacing: -1.4; // -0.13 em
-      text-margin: 7.7; // 0.7 em
-    }
-    [zoom >= 5] {
-      text-size: 12;
-      text-wrap-width: 45; // 3.8 em
-      text-line-spacing: -1.2; // -0.10 em
+    [way_pixels > 12000] {
```

Comment:
I think it would be a mistake to adjust the text labels based on way_pixel instead of by admin_level. This over-emphasizes areas at high latitudes (where way_pixels is 2 to 4 time larger than way_area at the equator) and over-emphasizes large provinces over small countries.